tools/capi_test/run_capi_application.bat

```diff
@@ -0,0 +1,25 @@
+set ONNX_MODEL_URL="https://media.githubusercontent.com/media/onnx/models/main/vision/classification/squeezenet/model/squeezenet1.0-7.onnx"
```

Comment:
Please try to not write more bat files. If any of the commands went wrong, you won't notice it. The pipeline will continue to run.

Reply:
Maybe .. you can try it.

Reply:
After failOnStandardError=true is added, some warnings seem being standard errors and make pipelines failed.
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=198395&view=logs&j=7ac7b520-9dc4-5344-f469-01b07bb067d1&t=74118d67-0ff0-5913-07dd-1ecec822b782
So here, we don't want to add failOnStandardError.
In this batch file, the last line of code is to run the application executable. If something goes wrong along the way, the executable can't be successfully created and the exit code  is not 0 which makes sure the Pipelines will fail.